src/ExpandObjects/epfilter.f90

```diff
@@ -15000,7 +14998,7 @@ SUBROUTINE CreateNewUnitary
   CALL AddToObjFld('Availability Schedule Name', base + uhpsHeatAvailSchedNameOff,'')
   CALL AddToObjFld('Rated Total Heating Capacity {W}', base + uhpsHPHeatCoilCapOff,'')
   CALL AddToObjFld('Rated COP', base + uhpsHPHeatCoilCOPOff,'')
-  CALL AddToObjFld('Rated Air Flow Rate {m3/s}', base + uhpsSupplyHeatFlowRateOff,'')
+  CALL AddToObjStr('Rated Air Flow Rate {m3/s}', 'autosize')
```

Comment:
Component flow rates/capacities should always be 'autosize'. User values flow into the sizing objects.